charts/clamav/values.schema.json

```diff
@@ -36,6 +36,16 @@
     "fullnameOverride": {
       "type": "string"
     },
+    "kind": {
+      "type": "string",
+      "enum": ["Deployment", "StatefulSet"]
+    },
+    "updateStrategy": {
```

Comment:
https://github.com/yannh/kubernetes-json-schema/blob/master/v1.31.0/statefulsetupdatestrategy.json

Reply:
You are absolutely right!

Reply:
I opted to only add properties of depth 1, since the complete schema is quite involved, and referencing external resources (especially in some personal Github account) seems wrong. Do tell if you disagree!

Reply:
all good!